loginsightwebhookdemo/bugzilla.py

```diff
@@ -27,13 +27,12 @@
 
 # Route without <ALERTID> are for LI, with are for vROps
 @app.route("/endpoint/bugzilla", methods=['POST'])
-@app.route("/endpoint/bugzilla/<ALERTID>", methods=['PUT'])
 @app.route("/endpoint/bugzilla/<TOKEN>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<TOKEN>/<ALERTID>", methods=['PUT'])
+@app.route("/endpoint/bugzilla/<TOKEN>/<ALERTID>", methods=['POST','PUT'])
 @app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['PUT'])
-@app.route("/endpoint/bugzilla/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['PUT'])
+@app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['POST','PUT'])
+@app.route("/endpoint/bugzilla2/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
```

Comment:
I do not see a way to support this. The option potential option is to wrap the URI with the path decorator, but this does not achieve the desired effect either. I suspect mosts/all will use the token option anyway (which is why it is the default) so I do not see any real impact to the proposed change. Thoughts?